bus-mapping/src/rpc.rs

```diff
@@ -121,15 +141,31 @@ impl<P: JsonRpcClient> GethClient<P> {
         Ok(resp.0.into_iter().map(|step| step.result).collect())
     }
 
-    /// Calls `eth_getProof` via JSON-RPC returning a [`EIP1186ProofResponse`]
-    /// returning the account and storage-values of the specified
-    /// account including the Merkle-proof.
+    /// Calls `eth_getCode` via JSON-RPC returning a contract code
+    pub async fn get_code_by_address(
+        &self,
+        contract_address: Address,
+    ) -> Result<Vec<u8>, Error> {
+        let address = serialize(&contract_address);
+        let id = serialize(&"0x2".to_string());
```

Comment:
Why is this "0x2" here?  This function should take a `BlockNumber` as argument and use it in the request.

Reply:
Thank you!
I fixed!